.changeset/cool-wombats-marry.md

```diff
@@ -0,0 +1,7 @@
+---
+"wrangler": patch
+---
+
+fix: the docs command should not crash if given search terms
```

Comment:
```suggestion
fix: the docs command should not error if given search terms
```

Reply:
Why this change? It is true that wrangler crashes and fails to do what you want it to do.
The error message you see is certainly not one we would want to show to users.
Are you concerned that crash has some other connotation?

Reply:
You're right. I wasn't I should have added right added it :)